docs/StardustDocs/topics/predicateJoin.md

```diff
@@ -0,0 +1,283 @@
+[//]: # (title: predicateJoin)
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.Join-->
+
+Joins two [`DataFrames`](DataFrame.md) by a join expression. 
+
+```kotlin
```

Comment:
code block contains non-kotlin code, so the highlighting goes wrong

Reply:
I think these blocks use pseudocode intentionally everywhere we have them. So should be ok to use here as well. As for highlighting, with `kotlin` at least valid parts are highlighted. Like interface :)

Reply:
you could also split it up in multiple blocks if you want pseudocode :)
Or do like I usually do:
```kotlin
// some pseudo code () -> Something
fun kotlinCode() {}
```

Reply:
i don't think that will work, you can see that even inside kotlin code there are symbols like [ ] for optional parts or | for enumeration, etc